### DISCUSSION TITLE
lesson 11 simplecollectible result wrong token

### DISCUSSION THREAD

**Discussion by gunomnifluencer** [smartcontractkit/full-blockchain-solidity-course-py#428] [Q&A] [answered]
Hello,

Today when I tried to deploy my NFT with **SimpleCollectible** I got returned link with token like this:
`https://testnets.opensea.io/assets/0x41ec3A3f4B9871b049a438F8e8159169BA606985/0`

when click and open it result 404 , this is not the first time, I had tried several time got same result, though the entire my nft has come in to testnet.opensea

and after I check on my profile at opensea, this is the working token link:
`https://testnets.opensea.io/assets/0xb8d686b2927639025a069363a1b89a852608ef2a/0`

Could somebody explain to me

Thanks

**Comment by cromewar** [comment, marked as answer]
Hello @gunomnifluencer I accessed to both links and I was able to see both actually, may be you got error 404 because the node on ipfs with the image was inaccessible at that moment so I would suggest you to try again.

![image](https://user-images.githubusercontent.com/20199180/143076600-015c37c4-fec8-4140-9955-6ca86d21408f.png)
![image](https://user-images.githubusercontent.com/20199180/143076615-26a47833-9e8f-498d-b936-67507878e104.png)

Anyways, the main difference between both is they belong to different wallet addresses as you can see on the images above, they are the exactly same (same image, and metada) as we are using the files Patrick showed up on the course, but they are minted on different wallets.

If you have more questions, feel free to ask.

Best regards, Cromewar.

**Reply by gunomnifluencer** [reply]
Hello,

Thanks for reply,

Just ask, is there other services besides **ipfs** to host nft?

because ipfs too slow

Many Thanks

**Reply by cromewar** [reply]
Well IPFS is not a service, is a protocol used by services like Pinata or Moralis, if you want alternatives to decentralized storage you can try with [FileCoin](https://filecoin.io/) or [storj](https://www.storj.io/) but I don't know if they give you URI for NFTS like IPFS.

Best regards.
